docs/contribute/e2eTesting.md

```diff
@@ -28,7 +28,10 @@ For more infos on Protractor see [Protractor API documentation](http://angular.g
   ```
   node ./node_modules/protractor/bin/webdriver-manager update
   ```
```

Comment:
I just think that this part here should first tell the users to use the npm commands or gulp tasks instead, becasue this ones will probably use the right pathes (at least they should) !!